app/javascript/components/student/tracks-list/StatusFilter.jsx

```diff
@@ -7,17 +7,21 @@ export function StatusFilter({ options, dispatch, ...props }) {
   }
 
   return (
-    <div>
+    <div className="status-filters">
       {options.map((option) => {
+        // TODO: Populate .count once these
+        // filters are moved client side
         return (
           <button
             key={option.value}
+            className={option.value === value ? 'selected' : ''}
```

Comment:
I considered checking for the absence of disabled, but decided they mean different things.